templating.rst

```diff
@@ -384,42 +339,20 @@ to render/extend ``app/Resources/views/base.html.twig``, you'll use the
 Referencing Templates in a Bundle
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-*If* you need to refer to a template that lives in a bundle, Symfony uses a **bundle**:**directory**:**filename**
-string syntax. This allows for several types of templates, each which lives in a
-specific location:
-
-* ``AcmeBlogBundle:Blog:index.html.twig``: This syntax is used to specify a
-  template for a specific page. The three parts of the string, each separated
-  by a colon (``:``), mean the following:
-
-  * ``AcmeBlogBundle``: (*bundle*) the template lives inside the AcmeBlogBundle
-    (e.g. ``src/Acme/BlogBundle``);
-
-  * ``Blog``: (*directory*) indicates that the template lives inside the
-    ``Blog`` subdirectory of ``Resources/views``;
+*If* you need to refer to a template that lives in a bundle, Symfony uses a
+``@bundle/Directory/filename.html.twig``. For example, suppose you have a bundle
+called ``AcmeBlogBundle``, then:
 
-  * ``index.html.twig``: (*filename*) the actual name of the file is
-    ``index.html.twig``.
-
-  Assuming that the AcmeBlogBundle lives at ``src/Acme/BlogBundle``, the
-  final path to the layout would be ``src/Acme/BlogBundle/Resources/views/Blog/index.html.twig``.
-
-* ``AcmeBlogBundle::layout.html.twig``: This syntax refers to a base template
-  that's specific to the AcmeBlogBundle. Since the middle, "directory", portion
-  is missing (e.g. ``Blog``), the template lives at
-  ``Resources/views/layout.html.twig`` inside AcmeBlogBundle. Yes, there are 2
-  colons in the middle of the string when the "controller" subdirectory part is
-  missing.
+* ``@AcmeBlog/Blog/index.html.twig``
+  Assuming that AcmeBlogBundle lives at ``src/Acme/BlogBundle``, the
+  final path to the file would be ``src/Acme/BlogBundle/Resources/views/Blog/index.html.twig``.
+  Notice that ``@AcmeBlog`` does **not contain the word Bundle** at the end - this
+  should be left off.
```

Comment:
My impression, is that the old `FooBundle:Default:index.html.twig` is out of style... and might even be removed someday (I might have just made that up... can't remember). And that the Twig namespaces are preferred. So, I thought - maybe we should just drop the old conversation, and just talk about the new Twig namespaces. There's less to explain, because the syntax is less weird.